sources/install/package_wordlists.sh

```diff
@@ -63,6 +63,16 @@ function install_seclists() {
     add-to-list "seclists,https://github.com/danielmiessler/SecLists,A collection of multiple types of lists used during security assessments"
 }
 
+function install_onelistforall() {
```

Comment:
Need to add the following line in the function
```bash
# CODE-CHECK-WHITELIST=add-aliases,add-history
```
Since you don't need those functions, this will avoid the CI/CD raising an issue